campaign.html

```diff
@@ -41,21 +41,22 @@
 </div>
 
 <div class="margin-top">
-  <h4>Thank you &hearts;</h4>
+  <h3>Thank you for letting us #WelcomeMoreWomenToTech</h3>
 </div>
 
-
-#donatebecause
-<div class="resize">
-  <img src="/img/blog/2016/donatebecause-collage.png">
-</div>
+<iframe src="https://player.vimeo.com/video/163633804" width="640" height="420" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+<p><a href="https://vimeo.com/163633804">RGSoC</a> from <a href="https://vimeo.com/user51331690">Rails Girls Summer of Code</a> on <a href="https://vimeo.com">Vimeo</a>.</p>
 
 <div class="margin-top">
   <p>
-    Rails Girls Summer of Code wants to offer three-months full time scholarships to
-    Open Source newcomers. Our aim is support <strong>24 students</strong>, which means 12 teams
-    &mdash; changing the future of Open Source one team at a time. With your help we can make it happen!
+
+    Rails Girls Summer of Code is a global program that provides full-time sponsored scholarships to its students for three months: from July 1st to September 30th. In three years, 104 students from 23 countries have been part of this project, having worked in 39 Open Source projects.
```

Comment:
The first sentence sounds good :) I'd rephrase the second to:

In the last three years, 104 students from 23 countries worked on a total of 39 Open Source projects through our initiative.